docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+---
+title: Submitting a pull request to Loopback 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
+---
+
+## How to submit a pull request (PR) for Loopback 4
+
+Here is a tutorial on how to submit a pull request (PR) for loopback v4.
+
+### 1. Log in to github
+
+Ensure you are logged in to your user account at <https://github.com>
+
+### 2. Fork the loopback-next repository
+
+In your browser, navigate to : <https://github.com/strongloop/loopback-next>  .
+
+Create your own fork of the repository, by pressing the `Fork` link on the right-hand side .
+
+![submit_pr_fork_main_repo.png](./imgs/submit_pr_fork_main_repo.png)
+
+Wait patiently as your fork is being created.
+
+When the forking process is complete, the repository will show up as `{your user id}/loopback-next`. In my case, it is `emonddr/loopback-next` .
+
+![submit_pr_my_forked_repo.png](./imgs/submit_pr_my_forked_repo.png)
+
+
+
+### 3. Create a feature branch for your work
+
+Notice your repo has a `master` branch already created ( refer bottom left corner of the picture above). It is commonplace to have this branch represent the latest, clean version of this repository's content. For the purposes of your PR, let's create a feature branch with a name indicative of your changes. In my case, it is `emonddr-doc-changes`.
+
+Click on the `down arrow` on the right-hand side of the `master` branch button.
+
+![submit_pr_create_feature_branch_1.png](./imgs/submit_pr_create_feature_branch_1.png)
+
+This brings up a small dialog for finding or creating a branch.
+
+![submit_pr_create_feature_branch_2.png](./imgs/submit_pr_create_feature_branch_2.png)
+
+Enter the `name` of your new `feature branch` in the search field. Because the branch `emonddr-doc-changes` doesn't yet exist, a menu item to create it appears below the search field. Click on the `Create branch` link.
```

Comment:
It would be much simpler to do so with `git` command:

```sh
git clone <your-fork-of-loopback-next>
cd loopback-next
git checkout -b <branch-name-for-your-pr>
```

Reply:
BTW, we don't have to repeat what's available in github docs - https://help.github.com/categories/collaborating-with-issues-and-pull-requests/. We can simply link to them if necessary.

Reply:
@raymondfeng , true...but I was asked in #2280 to give it the "flavour" of this article : https://www.nearform.com/blog/getting-into-open-source-for-the-first-time/ , which does go into some github detail. I figured this is what was desired. Also,  is it not more convenient to have the instructions in one place, instead of asking the user to open several other links? I do agree that we can provide a link to that awesome github document you provided in the References section.

Reply:
@raymondfeng , in my latest commit, I take the user
through these steps:

```
git clone <your-fork-of-loopback-next>
cd loopback-next
git checkout -b <branch-name-for-your-pr>
```
as per your suggestion.